recipe/0006-pycryptosat-win32-remove-runtime_library_dirs.patch

```diff
@@ -0,0 +1,23 @@
+--- python/setup.py.in
++++ python/setup.py.in
+@@ -129,8 +129,7 @@
+     extra_compile_args = cconf + ['-I${PROJECT_SOURCE_DIR}', '-I${PROJECT_BINARY_DIR}/cmsat5-src'],
+     extra_link_args = extra_link_args,
+     language = "c++",
+-    library_dirs=['.', '${PROJECT_BINARY_DIR}/lib'],
+-    runtime_library_dirs=['${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}'],
++    library_dirs=['.', '${PROJECT_BINARY_DIR}/lib', '${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}'],
```

Comment:
@msoos: It somehow didn't pick up the library from `${PROJECT_BINARY_DIR}/lib`.
For this build of `pycryptosat` we know that the library is already preinstalled with the `libcryptominisat` subpackage such that we can use `${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}` here.
You might have to investigate why it didn't work with `PROJECT_BINARY_DIR` to get a solution that works in the general case.